src/components/tree/data.js

```diff
@@ -0,0 +1,128 @@
+import { LoremIpsum } from 'lorem-ipsum';
+const lorem = new LoremIpsum({
+  wordsPerSentence: {
+    max: 8,
+    min: 2
+  }
+});
+
+let draggable = false;
+const rows = generateRows(randomNumber());
+
+export default {
+  headers: [
+    {
+      label: 'Title',
+      action: 'title-desc',
+      name: 'title'
+    },
+    {
+      label: 'Last Updated',
+      action: 'updated-at-desc',
+      labelIcon: 'calendar-icon',
+      name: 'updatedAt'
+    },
+    {
+      label: 'Published',
+      action: 'published-desc',
+      name: 'published',
+      labelIcon: 'circle-icon',
+      icon: 'circle-icon'
+    },
+    {
+      label: 'Link',
+      name: 'url',
+      icon: 'link-icon',
+      iconOnly: true
+    }
+  ],
+  rows,
+  draggable
+};
+
+function generateRows(number) {
+  const data = [];
+
+  for (let i = 0; i < number; i++) {
+    const row = generateRow();
+    data.push(row);
+  }
+
+  return data;
+}
+
+function generateRow(maxDepth = 5) {
+  const title = lorem.generateSentences(1);
+  const id = title.toLowerCase().replace(' ', '-');
+
+  const item = {
+    id,
+    title,
+    updatedAt: randomDay(),
+    published: randomBoolean() ? 'Published' : 'Unpublished',
+    url: `/${id}`,
+    // Even items without children need to have an `children` array eventually.
+    // This is to support dragging other items into them as children.
+    children: []
+  };
+
+  if (randomBoolean(0.3)) {
```

Comment:
Comment out this whole conditional to see how it works without nesting. The version history view is an example of that.